src/main/scala/mesosphere/marathon/tasks/MarathonTasks.scala

```diff
@@ -8,11 +8,26 @@ import org.apache.mesos.Protos.Attribute
 import scala.collection.JavaConverters._
 
 object MarathonTasks {
+
+  /*
+   * This implementation is somewhat problematic: it creates a task and pre-populates stagedAt with a timestamp close
+   * to the current system time, even though we did not yet receive a staged event from mesos. It is not clear, why.
+   * This might either be introduced to prevent tasks from being killed right away, or to be able to enhance the
+   * semantic of staged at – stagedAt could be understood as some sort of createdAt, when there is no status set.
+   * Either
+   * way, please be aware of the fact that there is a stagedAt timestamp even though we've not heard anything from mesos
+   * yet.
+   *
+   * The general problem seems to be that we don't have a timestamp of when the task was created – if stagedAt would
+   * default to 0, we couldn't measure how 'old' the task is. So, _maybe_ the solution would be to just add a createdAt
+   * property and set that when the task is created.
+   */
```

Comment:
I think that you can say that in fewer words. If you propose a different solution, I would rather create an issue for that.
 E.g.:

Despite its name, `stagedAt` is set on task creation and before the TASK_STAGED notification from Mesos. This is important because we periodically check for any tasks with an old `stagedAt` timestamp and kill them (See `KillOverdueStagedTasksActor`). If `stagedAt` remains 0 and this check is executed, the task will be killed immediately after being created.